pkg/log/zap/flags.go

```diff
@@ -26,6 +26,13 @@ import (
 	"k8s.io/klog"
 )
 
+const (
```

Comment:
@joelanford may not be better we use here the "github.com/sirupsen/logrus" levels definition instead of creating our own const/values as it was done [here](https://github.com/operator-framework/operator-sdk/blob/master/cmd/operator-sdk/bundle/internal/result.go#L93) for example?

Reply:
I believe these are int constants (https://github.com/sirupsen/logrus/blob/master/logrus.go#L96), string constants are directly inlined [line 25](https://github.com/sirupsen/logrus/blob/master/logrus.go#L25)

Reply:
We're also dealing with zap and logr levels here, not logrus. There don't appear to be string constants for the levels in zap or logr. I think what we're doing here in this PR is fine, and is in fact an improvement in that we now actually have constants rather than inlining them like we have been.